plugin/pkg/admission/initialization/initialization.go

```diff
@@ -208,6 +208,12 @@ func (i *initializer) Admit(a admission.Attributes) (err error) {
 		}
 		updated := accessor.GetInitializers()
 
+		// allow administrators to bypass initialization by setting an empty initializers struct
```

Comment:
I think the code change is correct but the comment is misleading-- In the create block above, the intent is to prevent the system from doing what it ordinarily would, i.e. adding in initializers. Here, we are just being nice and fixing objects for people.